src/sass/formio.form.builder.scss

```diff
@@ -151,7 +151,7 @@
 .builder-sidebar {
   &_scroll {
     position: sticky;
-    top: 30px;
+    top: 15px;
```

Comment:
This is very arbitrary change. We need to discuss how this will affect non-portal versions of the builder.

Reply:
It's just revert of this one https://github.com/formio/formio.js/pull/4855/commits/aac5b6d68cb2ed79d00e43da5149aee7d9c4f170